wqflask/wqflask/static/new/javascript/search_results.js

```diff
@@ -25,10 +25,10 @@ change_buttons = function(check_node = 0) {
 };
 
 $(function() {
-  var add, checked_traits, deselect_all, invert, remove, removed_traits, select_all;
+  var add, checked_traits, deselectAll, invert, selectAll;
```

Comment:
*nitpick:* `checked_traits` -> `checkedTraits`.

*issue:* For new JS code, let's try using `let` or `const` as opposed `var` since all our users will most definitely be using new browsers.  Consider:
```
{
var x = 1;
}
console.log(x)
```
you get: 1
vs 

```
{
var y = 1;
}
console.log(y)
```
you get: `undefined`

`var` is not block-scoped.  

You can read more [here](https://janakachathuranga.medium.com/why-we-use-const-and-avoid-var-3528ee550fb4)